webapp/src/utils.ts

```diff
@@ -442,6 +442,59 @@ class Utils {
             return block
         }
     }
+
+    static userAgent(): string {
+        return window.navigator.userAgent
+    }
+
+    static isDesktopApp(): boolean {
+        return Utils.userAgent().indexOf('Mattermost') !== -1 && Utils.userAgent().indexOf('Electron') !== -1
+    }
+
+    static getDesktopVersion(): string {
+        // use if the value window.desktop.version is not set yet
+        const regex = /Mattermost\/(\d+\.\d+\.\d+)/gm
+        const match = regex.exec(window.navigator.appVersion)?.[1] || ''
+        return match
+    }
+
+    /**
+     * Boolean function to check if a server version is greater than another.
+     *
+     * currentVersionParam: The server version being checked
+     * compareVersionParam: The version to compare the former version against
+     *
+     * eg.  currentVersionParam = 4.16.0, compareVersionParam = 4.17.0 returns false
+     *      currentVersionParam = 4.16.1, compareVersionParam = 4.16.1 returns true
+     */
+    static isServerVersionGreaterThanOrEqualTo(currentVersionParam: string, compareVersionParam: string): boolean {
```

Comment:
nit: you are only using it to compare the desktop app version, and the name here is misleading. probably something like `isVersionGreaterThanOrEqualTo` would be better. The comments also are misleading.

Reply:
Done.